frontend/src/component/UIFactory.tsx

```diff
@@ -109,6 +110,10 @@ export class UIFactory {
     );
   }
 
+  public createAdminPage(): ReactElement {
```

Comment:
I recommend doing the followings instead:

## For this PR:
1. Add `includeAdminPanel` in here: https://github.com/short-d/short/blob/master/frontend/src/service/feature-decision/FeatureDecision.service.ts. 
1. Modify https://github.com/short-d/short/blob/f0c1c419f5a4adfaaa30e68cd152bb890c93346a/frontend/src/service/HTTP.service.ts#L1:18 to allow passing HTTP headers.
1. Include authentication header: `Authorization: bearer token` here: https://github.com/short-d/short/blob/f0c1c419f5a4adfaaa30e68cd152bb890c93346a/frontend/src/service/ShortHTTP.api.ts#L14:3
1. ExPAND https://github.com/short-d/short/blob/master/backend/app/usecase/feature/dynamic.go#L18 to pass `user` struct.

## Next
We will design permission toggle together. I recommend reading this article fully before start feature toggle discussion: https://martinfowler.com/articles/feature-toggles.html